### DISCUSSION TITLE
EMTEC E-240 Preset v1

### DISCUSSION THREAD

**Discussion by VexorionReal** [ntsc-rs/ntsc-rs#142] [Presets]
This preset is, and at least attempts to be, based on the EMTEC E-240 VHS tape (although I'm unsure whether it's Home TV Master or Standard Quality)

![ELIZE - AUTOMATIC (I'M TALKING TO YOU) avi_snapshot_01 06 091_pal](https://github.com/user-attachments/assets/55354ebb-6704-4988-b462-bf55cfbf8f74)

[EMTEC E-240 Preset v1.json](https://github.com/user-attachments/files/16444514/EMTEC.E-240.Preset.v1.json)

**Comment by VexorionReal** [comment]
@Degamisu What do you mean by "what camera"?

**Reply by Degamisu** [reply]
Ignore the comment, I thought you took the video. My mistake!

**Comment by oisintheblue** [comment]
Cool!

**Comment by VexorionReal** [comment]
@smfthe1199 Add this to the preset pack if you can.

**Reply by smfthe1199** [reply]
Don't worry, it's gonna be released soon, stay tuned.

UPDATE: The updated version of the "NTSC-RT Ultimate Preset Pack" **has now released.** See #157.
